app/models/item_action.js

```diff
@@ -26,5 +26,32 @@ export default DS.Model.extend({
 
   user: belongsTo("user", {
     async: false
+  }),
+
+  absoluteQuantity: Ember.computed("quantity", function() {
+    return Math.abs(this.get("quantity"));
+  }),
+
+  locationSubscript: Ember.computed("quantity", function() {
+    return Number(this.get("quantity")) >= 0 ? "To" : "From";
+  }),
+
+  sourceSubscript: Ember.computed("quantity", function() {
+    return Number(this.get("quantity")) >= 0 ? "From" : "To";
+  }),
+
+  arrow: Ember.computed("quantity", function() {
+    return Number(this.get("quantity")) >= 0 ? "arrow-up" : "arrow-down";
+  }),
```

Comment:
Those probably shouldn’t be in the model, we should keep it data-centric. This feels more like UI related, maybe a helper for it ?